### PR TITLE
Restore CI in offline mode

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: ci

**Context**:
We found some bug lately (#3120), not catched by the CI, because we're no longer testing in offline environments since #1928.

**Summary**:
Restore the offline mode by configuring some security groups on Openstack and using a proxy for the YUM repositories.

**Acceptance criteria**:

### CODE REVIEW

File: eve/workers/openstack-terraform/terraform/common.tf [hunk 1]
```diff
@@ -23,6 +23,21 @@ variable "nodes_count" {
...
 }
 
+variable "offline" {
+  type    = bool
+  default = true
+}
+
+variable "proxy_ip" {
+  type    = string
+  default = "10.100.4.67"
```
Comment (TeddyAndrieux) [inline eve/workers/openstack-terraform/terraform/common.tf]: We cannot rely on ScalityCloud dns `proxy-cache` ? IP is mandatory for security group rules right ?

**Review by TeddyAndrieux — COMMENTED** [review]
LGTM

It seems RHEL8 does not work offline (don't know why just build status)

File: eve/workers/openstack-terraform/terraform/common.tf (hunk 1 above)
Comment (gdemonet) [reply eve/workers/openstack-terraform/terraform/common.tf]: :+1: - the `dns_a_record_set` data source cannot resolve it, as you did for `subscription.rhsm.redhat.com`?

Comment (alexandre-allard) [reply eve/workers/openstack-terraform/terraform/common.tf]: Yes, sure, we could totally do that.

File: eve/main.yml [hunk 2]
```diff
@@ -1641,6 +1643,14 @@ stages:
...
           property: os
           value: "rhel-8"
+      - SetProperty:
+          name: Set offline property to false
+          property: offline
+          value: false
+      - SetProperty:
+          name: Set use_proxy property to false
+          property: use_proxy
+          value: false
```
Comment (gdemonet) [inline eve/main.yml]: Worth adding a comment + link to a ticket (not sure if we've created one yet) to remember why this is disabled and what we should do to re-enable it.

**Review by gdemonet — COMMENTED** [review]
Except for a small comment on disabling offline for RHEL 8, I think this is good to merge :slightly_smiling_face: 
Just wondering, how does this work with already existing snapshots which were not configured to run offline? (I guess it still works, but worth checking)

**Review by gdemonet — APPROVED** [review]

**Review by gdemonet — DISMISSED** [review]

**Review by gdemonet — APPROVED** [review]

**Pull request merged**